### ISSUE TITLE
InAppMessageListener: onMessageFinished: resolutionInfo Data not accessible Version 9.1.0

### ISSUE THREAD

**Issue by c-schone** [urbanairship/android-library#78]
Hello,

I'm trying to access the resolutionInfo in the onMessageFinished method which doesn't work.

I can't access the type nor the buttonInfo nor the displayMilliseconds.

It seems like a bug to me as I don't think you would provide a resolutionInfo for a listener event which fields can't be accessed at all.

A little Code snippet to make my goal more understandable:

```
inAppMessageManager.addListener(new InAppMessageListener() {
            @Override
            public void onMessageDisplayed(@NonNull final String s, @NonNull final InAppMessage inAppMessage) {
//Not interested
            }

            @Override
            public void onMessageFinished(@NonNull final String s, @NonNull final InAppMessage inAppMessage, @NonNull final ResolutionInfo resolutionInfo) {
//Here is where I want to access resolutionInfo.type !
            }
        });
```

I want to achieve to get to know if the user clicked on the InAppMessage.

**Comment by c-schone** [comment]
Alright, thank you for clarification.

**Comment by rlepinski** [comment]
Fixed in 9.2.0
